ports/rp2/CMakeLists.txt

```diff
@@ -199,6 +199,10 @@ if (MICROPY_PY_LWIP)
     target_compile_definitions(${MICROPY_TARGET} PRIVATE
         MICROPY_PY_LWIP=1
     )
+else()
+	target_compile_definitions(${MICROPY_TARGET} PRIVATE
+        MICROPY_PY_LWIP=0
```

Comment:
This shouldn't be necessary. The default (unset) is fine.

Reply:
Okey, I delete the else statement.